src/pages/iou/IOUModal.js

```diff
@@ -410,8 +410,8 @@ class IOUModal extends Component {
         const currentStep = this.steps[this.state.currentStepIndex];
         const reportID = lodashGet(this.props, 'route.params.reportID', '');
         return (
-            <ScreenWrapper>
-                {({didScreenTransitionEnd}) => (
+            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+                {({didScreenTransitionEnd, safeAreaPaddingBottomStyle}) => (
```

Comment:
Same as above - Why does this need the `safeAreaPaddingBottomStyle` if `includeSafeAreaPaddingBottom` is false?

Reply:
Different reason from above. Some of the screens in the IOU flow need the safe area padding and some do not. If we apply it at the `ScreenWrapper` level then we can't customize which screens have the safe area padding and which do not.